src/util/report.py

```diff
@@ -286,6 +286,12 @@ def ref(xid):
     except Exception:
         _logger.warning("Cannot announce message", exc_info=True)
 
+    # Chat window with the report will be open post-upgrade for the admin user
+    if not version_gte("saas~18.2"):
```

Comment:
`fold_state` appears in v9
```suggestion
    if version_between("9.0", "saas~18.2"):
```